exercises/custom_rock_paper_scissors.livemd

```diff
@@ -25,15 +40,69 @@ For example,
 game = CustomGame.new(:grass, :fire, :water)
 
 CustomGame.play(game, :fire, :water)
-":water beats :fire"
+"water beats fire"
```

Comment:
I think the atom should be included (based on the tests).

I'm happy to alter it, however we should reflect the tests to match.

Reply:
Well, when we interpolate an atom in a string it removes the colon

Reply:
TIL. Approving!